### PR TITLE
Form controls and inputs now have basic focus styles

### PR DESCRIPTION
**Pull request by rdjpalmer**

### CODE REVIEW

File: components/Checkbox/Checkbox.css [hunk 1]
```diff
@@ -50,6 +50,10 @@
   background: var(--color-white);
 }
 
+.input:focus + .label:before {
+  box-shadow: 0 0 3px 1px var(--color-blue);
```
Comment (iest) [inline components/Checkbox/Checkbox.css]: The focus colour we use on `FormInput` is the teal. We should keep them consistent — either way.

File: components/FormInput/FormInput.css [hunk 2]
```diff
@@ -101,6 +102,11 @@
 /* Borderless */
 .borderless {
   box-shadow: none;
+  transition: all 300ms ease;
+  border: 1px solid transparent;
+}
+.borderless.containerFocus.labelInside {
+  border-bottom-color: var(--color-green);
```
Comment (iest) [inline components/FormInput/FormInput.css]: What does this look like in practise?

File: components/Checkbox/Checkbox.css (hunk 1 above)
Comment (rdjpalmer) [inline components/Checkbox/Checkbox.css]: More inclined to move the `FormInput` colour to blue, for cases where we haven't overridden the browser default. Both `Checkbox` and `Radio` have a heavy use of a primary colour and teal would be too similar also.

File: components/FormInput/FormInput.css (hunk 2 above)
Comment (rdjpalmer) [inline components/FormInput/FormInput.css]: screen shot 2015-10-23 at 17 28 20

File: components/Checkbox/Checkbox.css (hunk 1 above)
Comment (iest) [inline components/Checkbox/Checkbox.css]: @scott-riley wat do

Comment (rdjpalmer) [inline components/Checkbox/Checkbox.css]: FYI `Radio` with blue looks like so:

screen shot 2015-10-23 at 16 13 54

**Comment by iest** [comment]
Let's get this merged. We can always come back to it.

:ship: :it:

**Pull request merged**